index.html

```diff
@@ -25,7 +26,121 @@ <h1 class="header2">I'm Ali<br>Glad to see<br>you!</h1>
+                        <li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+                        <li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+                    </ul>
+                    <a class="see_project" href="https://"><img  src="img/see_project.png" alt="see projects"></a>
+                </section>
+                <section class="tonic_border">
+                    <section class="tonic_project">
+                        <a class="tonic_pic" href="https://"><img src="img/port2.png" alt=""></a>
+                        <p class="tonic_name">Multi-Post<br>Stories</p>
+                    </section>
+                        <ul class="info2">
+                            <li class="canopy">CANOPY</li>
+                            <li class="counter1"><img src="img/Counter.png" alt=""></li>
+                            <li class="backend">BACK END DEV</li>
+                            <li class="counter2"><img src="img/Counter.png" alt=""></li>
+                            <li class="year">2015</li>
+                        </ul>
+                        <p class="description">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required.</p>
+                        <ul class="programs">
+                            <li class="html"><a href="https://"><img src="img/html.png" alt=""></a></li>
+                            <li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+                            <li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+                        </ul>
+                        <a class="see_project" href="https://"><img  src="img/see_project.png" alt="see projects"></a>
+                    </section>
+        <section class="main_body">
+            <h1 class="header2">About<br>Myself</h1>
+            <p class="body2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">Let's Connect</p>
+            <ul class="icons">
+                <li class="twitter"><a href="https://"><img src="img/Icon - Twitter.png" alt="Twitter"></a></li>
+                <li class="linkedin"><a href="https://"><img src="img/Icon - Linkedin.png" alt="Linkedin"></a></li>
+                <li class="medium"><a href="https://"><img src="img/Icon - Medium.png" alt="Medium"></a></li>
+                <li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
+                <li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
+            </ul>
+            <a class="see_project" href="https://"><img  src="img/get_resume.png" alt="see projects"></a>
```

Comment:
- [ ] The link here is not visible. 
**design**
![see](https://user-images.githubusercontent.com/99656687/219884442-4b451680-e42d-4be0-a755-6d3cef473737.png)

**implementation**
![see2](https://user-images.githubusercontent.com/99656687/219884493-e725d0d8-b73e-4ed3-9cc3-8d69941d5014.png)